.doc_gen/metadata/services.yaml

```diff
@@ -409,6 +409,21 @@ eventbridge:
   tags:
     product_categories: {'Application Integration'}
   version: eventbridge-2015-10-07
+firehose:
+  long: "&FHLong;"
+  short: "&FH;"
+  sort: Firehose
```

Comment:
`sort: Kinesis Data Firehose`
(sort name is the short name minus any "AWS" or "Amazon")